.php-cs-fixer.dist.php

```diff
@@ -40,6 +40,9 @@
                 ['Serializer\\*', 'VirtualProperty', 'Accessor', 'Type', 'Groups', 'Expose', 'Exclude', 'SerializedName', 'Inline', 'ExclusionPolicy'],
             ],
         ],
+        'get_class_to_class_keyword' => false, // should be enabled as soon as support for php < 8 is dropped
```

Comment:
@Prokyonn stumbled over issues with this rule for PHP < 8.0 we can remove this in 2.5